doc/common/openstack-release.md

```diff
@@ -1,32 +1,32 @@
-# CNTT OpenStack Baseline Release Selection
+# Anuket OpenStack Baseline Release Selection
 
-This section specifies policies for the selection of the next CNTT OpenStack baseline release and the number of releases that CNTT shall support:
+This section specifies policies for the selection of the next Anuket OpenStack baseline release and the number of releases that the Anuket specifications shall support:
 -	criteria for the triggering of the next baseline selection
--	criteria to use in choosing the next OpenStack release, and 
--	the number of OpenStack releases to be supported by CNTT
+-	criteria to use in choosing the next OpenStack release, and
+-	the number of OpenStack releases to be supported by Anuket specifications
```

Comment:
Can we please not make changes

Reply:
As Anuket contains subprojects other than the former CNTT, like Functest or Airship Installer, changing CNTT to Anuket would be misleading, as @tomkivlin pointed out in #2362.